Update javascript.html.markdown (#3873)
adambard/learnxinyminutes-docs: javascript.html.markdown

// ES6 Additions

// The "let" keyword allows you to define variables in a block scope, 

> // as opposed to an only functional scope like the var keyword does.

Reviewer: ```suggestion
// as opposed to a functional scope like the "var" keyword does.
```
don't really need to stress on it :smile:
